packages/ui/src/Text/index.tsx

```diff
@@ -102,7 +102,9 @@ const getTextOptionClasses = ({
   break: breakProp,
   whitespace,
 }: TextProps): string => {
-  const truncateClass = truncate ? cn('truncate') : '';
+  const truncateClass = truncate
+    ? cn('truncate max-w-[150px] overflow-hidden whitespace-nowrap')
```

Comment:
issue: the `max-w-[150px]` class would break many of existing components. Truncation width is generally defined in the parent component.

also, `truncate` class applies both `overflow-hidden` and `whitespace-nowrap` under the hood

Reply:
I’ll move the truncation width to the parent component. Moving forward, I’ll focus on applying styling at the parent level where possible to avoid introducing breaking changes to existing components.